curvefs/src/client/dentry_cache_manager.cpp

```diff
@@ -155,6 +155,11 @@ CURVEFS_ERROR DentryCacheManagerImpl::ListDentry(uint64_t parent,
         }
     } while (perceed);
 
+    for (const auto &it : *dentryList) {
+        std::string key = GetDentryCacheKey(it.parentinodeid(), it.name());
+        dCache_->Put(key, it);
```

Comment:
I don't think it is a good idea, putting all dentry of ListDentry in dCache will result in lower cache hit rate.

Reply:
Whether it will reduce the cache hit rate is not easy to say, not modify this time and can do some test later.